.github/workflows

```diff
@@ -0,0 +1 @@
+linters.yml
```

Comment:
- [ ] As the previous reviewer has earlier suggested, kindly set up your GitHub workflow for linters properly. 
The folder path should be ` .github/workflows/linters.yml` after which you copy-paste the linter configurations into the linters.yml file 👍🏼 .

Kindly follow the steps for setting up [linters ](https://github.com/microverseinc/linters-config/tree/master/html-css) closely